python/semantic_kernel/ai/open_ai/services/azure_chat_completion.py

```diff
@@ -7,38 +7,53 @@
 from semantic_kernel.ai.open_ai.services.open_ai_chat_completion import (
     OpenAIChatCompletion,
 )
+from semantic_kernel.utils.notebooks import try_get_api_info_from_synapse_mlflow
 
 
 class AzureChatCompletion(OpenAIChatCompletion):
     _endpoint: str
     _api_version: str
+    _api_type: str
 
     def __init__(
         self,
         deployment_name: str,
-        endpoint: str,
-        api_key: str,
+        endpoint: Optional[str] = None,
+        api_key: Optional[str] = None,
         api_version: str = "2023-03-15-preview",
         logger: Optional[Logger] = None,
+        use_ad_auth=False,
     ) -> None:
+        if use_ad_auth and (endpoint is None or api_key is None):
+            # Try to get endpoint/api_key via alternative methods
+            results = try_get_api_info_from_synapse_mlflow()
+            if results is not None:
+                endpoint, api_key = results
+            else:
+                raise ValueError(
```

Comment:
suggestion: raise ValueError("Azure AD authentication failed. Alternatively provide an endpoint and API key.")

Reply:
Error message and tests updated